config.py

```diff
@@ -16,6 +16,8 @@ class Config:
     DB_PASSWORD = os.getenv("DB_PASSWORD")
     DB_NAME = os.getenv("DB_NAME")
 
+    JWT_SECRET_KEY = os.getenv("SECRET_KEY")
```

Comment:
Tengo entendido que las claves de JWT deben seguir cierta signatura por defecto. Por lo cual no son necesariamente compatibles con cualquier clave secreta nativa de Flask. Deberíamos usar variables de entorno distintas para evitar problemas. Habría que agregar esa nueva variable al archivo de variables de entorno de ejemplo.

```suggestion
    JWT_SECRET_KEY = os.getenv("JWT_SECRET_KEY")
```